packages/datagrid/src/components/PlaygroundCellEditor/ApplyToIdenticalValues.component.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+	ButtonPrimary,
+	ButtonSecondary,
+	Form,
+	StackHorizontal,
+	StackVertical,
+} from '@talend/design-system';
+
+import theme from './ApplyToIdenticalValues.scss';
+
+interface ApplyToIdenticalValuesPropTypes {
```

Comment:
If it can help: https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases

TLDR; interfaces are the way to go by default, and types comes to rescue for specific cases.